path_metadata.go

```diff
@@ -136,11 +145,69 @@ func (b *versionedKVBackend) pathMetadataRead() framework.OperationFunc {
 				"max_versions":         meta.MaxVersions,
 				"cas_required":         meta.CasRequired,
 				"delete_version_after": deleteVersionAfter.String(),
+				"custom_metadata":      meta.CustomMetadata,
 			},
 		}, nil
 	}
 }
 
+const maxCustomMetadataKeys = 64
+const maxCustomMetadataKeyLength = 128
+const maxCustomMetadataValueLength = 512
```

Comment:
Curious to know if these numbers are coming from a reasonable guess or if it is driven by any reference.

Reply:
Great. I guess you will be adding entries for these as well in that page :-)

Reply:
Yup

https://github.com/hashicorp/vault/blob/71d9cdd4d4fbc15a498d2c3a701457556f3219c2/website/content/docs/internals/limits.mdx#versioned-key-value-store-kv-v2-secret-engine